edb/schema/indexes.py

```diff
@@ -1694,6 +1702,14 @@ def is_fts_index(
     return index.issubclass(schema, fts_index)
 
 
+def get_fts_index_id(
+    schema: s_schema.Schema,
+    index: Index,
+) -> str:
+    # TODO: Use the model name?
+    return f'base'
```

Comment:
Hmm. This is OK for 5.x since we prohibit more than one index, but ideally the name should be derived from index identity (perhaps just use DDL identity?).  Model name is not enough, because we may want to allow multiple indexes with different index parameters using the same model to generate embeddings.  Semantic search is very fiddly with respect to the kind of index and its parameters (there are 6 permutations of main params + free standing iffvlat/hnsw params).

Reply:
To clarify: index (and the embedding col) selection would happen based on `ai::search` arguments.